torch/special/__init__.py

```diff
@@ -799,3 +794,65 @@
     tensor([1., 1., 1.])
 
 """.format(**common_args))
+
+
+def _logsumexp_return_sign_false(
+        input: Tensor, dim: typing.List[int], keepdim: bool = False,
+        b: typing.Optional[Tensor] = None, return_sign: bool = False, out: Tensor = None) -> Tensor:
+    return _special.special_logsumexp(input, dim, keepdim, b, out=out)
+
+
+def _logsumexp_return_sign_true(
+        input: Tensor, dim: typing.List[int], keepdim: bool = False,
+        b: typing.Optional[Tensor] = None, return_sign: bool = False,
```

Comment:
Kwarg-only logic will need to be here, too, I think

Reply:
I've implemented that on my working branch (not yet pushed). But that leads to another JIT issue:

```
torch.jit.frontend.NotSupportedError: Compiled functions can't take variable number of arguments or use keyword-only arguments with defaults:
  File "/Users/till/git/pytorch/torch/special/__init__.py", line 801
def _logsumexp_return_sign_false(
        input: Tensor, dim: typing.List[int], keepdim: bool = False, *,
        b: typing.Optional[Tensor] = None, return_sign: bool = False, out: Tensor = None) -> Tensor:
                                     ~~~~ <--- HERE
    return _special.special_logsumexp(input, dim, keepdim, b=b, out=out)
```